logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/register/RegisterAccountUseCase.kt

```diff
@@ -47,7 +47,7 @@ class RegisterAccountUseCase(
 }
 
 sealed class RegisterResult {
-    class Success(val value: RegisterResponse) : RegisterResult()
+    class Success(val value: UserDTO) : RegisterResult()
```

Comment:
Just food for thought:

I wonder if exposing `network` classes through `logic` doesn't come without down-sides.

I think exposing only as an output is ok. But we gotta be careful and never require internal modules classes as inputs. As it will assume that the consumer already has access to these libs.

`RegisterResponse` was from `network` already. So I guess that for this PR, this is alright.

Reply:
This will change in my next PR since Register will return `User` + `AuthSession` (hint login user after register)